hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestRootedOzoneFileSystem.java

```diff
@@ -191,7 +198,7 @@ public static Path getBucketPath() {
   }
 
   @Rule
-  public Timeout globalTimeout = Timeout.seconds(300);
+  public Timeout globalTimeout = Timeout.seconds(3000);
```

Comment:
Was this change meant for debugging? `3000s` is way too long to be meaningful here IMO.

Reply:
Left over from testing.  Corrected.  Thanks!